### PR TITLE
Set recipient to atoms in addScipionAttribute()

### PR DESCRIPTION
**Pull request by csimong**
Set recipient param to "atoms" when creating scipion attribute with addScipionAttribute to avoid missing data.

### CODE REVIEW

**Review by DavidHerreros — APPROVED** [review]

**Comment by csimong** [comment]
I thought git would merge this PR after the reviewers' approvals. 
I just wanted to let you know that I do it now.

**Pull request merged**
